tests/lighthouse/evaluate.py

```diff
@@ -0,0 +1,24 @@
+import sys
```

Comment:
Can we replace this file by using the lighthouse config with the "assert" property? See this example: https://mikemfleming.com/blog/integrating-lighthouse-ci-with-bitbucket-pipelines/

This would be cleaner as it would replace this custom solution with the lighthouse native one.

Reply:
This file should be deleted